templates/admin/job_template/index.html.ep

```diff
@@ -84,9 +84,7 @@
                 <p id="loading"><i class="fa fa-spinner fa-spin"></i> Loading…</p>
 
                 <select id="machines-template" multiple="true"
```

Comment:
It would be nice to make this one line (as well as the other occurrence).

Reply:
done